api/pyproject.toml

```diff
@@ -20,6 +20,7 @@ pydantic = "^2.5.3"
 sqlalchemy = "^2.0.25"
 psycopg2-binary = "^2.9.9"
 cryptography = "^42.0.4"
+python-multipart = "^0.0.9"
```

Comment:
Where are we using multi-part?

Reply:
We need it for the /index-data endpoint as the Form data requires "python-multipart" to be installed.

Reply:
I think newer version of fastapi doesn't require to install it separately. Please confirm this.

Reply:
Yeah its not needed after fastapi version update.